src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -908,6 +1036,117 @@ public void setNumOfRubricSubQuestions(int numOfRubricSubQuestions) {
     }
 
     /**
+     * Class to store any stats related to a recipient.
+     */
+    private class RubricRecipientStatistics {
+        String recipient;
```

Comment:
Shouldn't this be called `recipientEmail`? From the way the CSV is constructed below, it seems that this is the case.